src/openrct2/actions/TrackRemoveAction.hpp

```diff
@@ -216,7 +216,7 @@ DEFINE_GAME_ACTION(TrackRemoveAction, GAME_COMMAND_REMOVE_TRACK, GameActionResul
 
             if (entranceDirections & TRACK_SEQUENCE_FLAG_ORIGIN && (tileElement->AsTrack()->GetSequenceIndex() == 0))
             {
-                if (!track_remove_station_element(mapLoc.x, mapLoc.y, mapLoc.z / 8, _origin.direction, rideIndex, 0))
+                if (!track_remove_station_element({ mapLoc.x, mapLoc.y, mapLoc.z / 8, _origin.direction }, rideIndex, 0))
```

Comment:
CoordsXYZ should never have a /8 on its Z component. You need to go through and remove the multiplies and divides of the z component. The /8 means that this was a smallZ function we want the whole codebase to be in bigZ

Reply:
@duncanspumpkin Will do, thanks for the clarification on smallZ vs bigZ